### PR TITLE
Update BUMP yaml keys

### PR DESCRIPTION
**Pull request by benjaminmenetrier**
Companion PR of https://github.com/JCSDA-internal/saber/pull/389

To update all the yamls:
```
cd ${SOCA}/ewok/defaults
for file in `ls *.yaml`; do ${SABER}/tools/update_yaml.py $file; done
cd ${SOCA}/test/testinput
for file in `ls *.yml`; do ${SABER}/tools/update_yaml.py $file; done
cd ${SOCA}/tutorial/config
for file in `ls *.yaml`; do ${SABER}/tools/update_yaml.py $file; done
```

### CODE REVIEW

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Review by kbhargava — APPROVED** [review]
Looks good. tested on orion.

**Comment by benjaminmenetrier** [comment]
This PR can be merged now.

File: tutorial/config/staticb.yaml [hunk 1]
```diff
@@ -18,15 +18,16 @@ background error:
   analysis variables: *ana_vars
   date: *date
   bump:
-    verbosity: main
```
Comment (travissluka) [inline tutorial/config/staticb.yaml:21]: @benjaminmenetrier I just now noticed that we are getting bump output duplicated per PE in our tests, which makes it hard to read. Is there anyway to keep that from happening as was the case when we had `verbosity: main` ?

Comment (benjaminmenetrier) [reply tutorial/config/staticb.yaml:21]: Hi @travissluka, could you try to run SOCA with the branch `feature/missing_channels` for SABER and SOCA? I think it should fix your issue. Thanks!

Comment (travissluka) [reply tutorial/config/staticb.yaml:21]: yes, thank you @benjaminmenetrier, that seems to fix things!

**Pull request merged**